hmc-cradlepoint/cradlepoint
# Mongodb setup

to test: 
1) pull branch
2) run npm i
add a new file in the cradlepoint directory called .env.local and copy the following into it: MONGODB_URI=mongodb://admin:correcthorsebatteryalien@c35.clinic.cs.hmc.edu
MONGODB_DB=cradlepoint
3) start server using npm run dev
4) go to http://localhost:3000/
5) Verify that the page says connected to mongodb
if yes, then app can connect to mongodb as desired
